packages/EasyErrorHandler/docs/config.md

```diff
@@ -17,27 +17,27 @@ called `easy_error_handler`.
 
```

Comment:
Let's add an info about how the package works with different error code sources.

Reply:
Added